apps/aens/src/aens_transfer_tx.erl

```diff
@@ -181,19 +193,22 @@ for_client(#ns_transfer_tx{nonce             = Nonce,
                            fee               = Fee,
                            ttl               = TTL} = Tx) ->
     #{<<"vsn">>              => version(),
-      <<"account">>          => aec_base58c:encode(account_pubkey, account(Tx)),
+      <<"account">>          => aec_base58c:encode(id_hash, account(Tx)),
       <<"nonce">>            => Nonce,
-      <<"name_hash">>        => aec_base58c:encode(name, name_hash(Tx)),
-      <<"recipient_pubkey">> => aec_base58c:encode(account_pubkey, recipient_account(Tx)),
+      <<"name_hash">>        => aec_base58c:encode(id_hash, name(Tx)),
+      <<"recipient_pubkey">> => aec_base58c:encode(id_hash, recipient(Tx)),
       <<"fee">>              => Fee,
       <<"ttl">>              => TTL}.
 
 %%%===================================================================
 %%% Getters
 %%%===================================================================
 
--spec recipient_account(tx()) -> aec_keys:pubkey().
-recipient_account(#ns_transfer_tx{recipient_account = Recipient}) ->
+recipient(#ns_transfer_tx{recipient_account = Recipient}) ->
+    Recipient.
+
+recipient_pubkey(#ns_transfer_tx{recipient_account = Recipient}) ->
+    %% TODO: Resolve name
```

Comment:
When enabling name transfer receiver to be name, the Swagger param [`recipient_pubkey`](https://github.com/aeternity/epoch/blob/5a72c277eb0f8619d3c7f8af5c94e63263db10df/config/swagger.yaml#L2727) shall probably be renamed as name is (?) expressed as hash hence not pubkey.